File: PaMVPDemo/lib-core/src/main/java/com/pa/core/base/BaseModel.java
package com.pa.core.base;

import com.pa.core.nethelper.RetrofitManger;

import retrofit2.Retrofit;

public class BaseModel implements IModel {

    protected Retrofit mRetrofit;

    public BaseModel(){
        mRetrofit = RetrofitManger.getInstance().createRetrofit();
    }
}

File: PaMVPKotlinDemo/lib-core/src/main/java/com/pa/core/base/BaseModel.kt
package com.pa.core.base

import com.pa.core.nethelper.RetrofitManger

import retrofit2.Retrofit

open class BaseModel : IModel {

    protected var mRetrofit: Retrofit? = null

    init {
        mRetrofit = RetrofitManger.getInstance()?.createRetrofit()
    }
}
